app/models/membership.rb

```diff
@@ -19,4 +19,8 @@ def approve!
     return if status == 'member'
     update(status: 'member', status_updated_at: Time.current)
   end
+
+  def set_moderator!
```

Comment:
Not blocking: `set` is a generic verb which makes it slightly less clear waht this method does. Perhaps we want a more specific verb such as "appoint" or something? https://www.thesaurus.com/browse/set